packages/client-twitter/src/post.ts

```diff
@@ -185,11 +185,13 @@ export class TwitterPostClient {
         } else {
             elizaLogger.log("Action processing loop disabled by configuration");
         }
+        generateNewTweetLoop();
```

Comment:
I added this piece of code, without this tweets won't generate into twitter
```generateNewTweetLoop();```

Reply:
I caught that and PR'd that earlier too

Reply:
Got it